app/assets/stylesheets/application.css.scss

```diff
@@ -54,6 +54,11 @@ table {
   width: 100%;
 }
 
+.table-striped>tbody>tr:nth-child(odd)>td,
+.table-striped>tbody>tr:nth-child(odd)>th {
+   background-color: #263645;
```

Comment:
Color literals like `#263645` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 3 spaces